addons/portal/wizard/portal_wizard.py

```diff
@@ -196,7 +177,8 @@ def _retrieve_user(self, cr, uid, wizard_user, context=None):
         res_users = self.pool.get('res.users')
         domain = [('partner_id', '=', wizard_user.partner_id.id)]
         user_ids = res_users.search(cr, uid, domain, context=context)
-        return user_ids and res_users.browse(cr, uid, user_ids[0], context=context) or False
+        if user_ids:
+            return wizard_user.write({'user_id': user_ids[0]})
```

Comment:
return in one line like : 
return user_ids and wizard_user.write({'user_id': user_ids[0]}) or False